runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -0,0 +1,110 @@
+package net.runelite.client.plugins.raids;
+
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Binder;
+import com.google.inject.Provides;
+import java.awt.Color;
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import javax.inject.Inject;
+import net.runelite.api.ChatMessageType;
+import net.runelite.api.Client;
+import net.runelite.api.events.ChatMessage;
+import net.runelite.api.events.ConfigChanged;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.chat.ChatColor;
+import net.runelite.client.chat.ChatColorType;
+import net.runelite.client.chat.ChatMessageBuilder;
+import net.runelite.client.chat.ChatMessageManager;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+
+@PluginDescriptor(
+		name = "Raids Plugin"
+)
+public class RaidsPlugin extends Plugin
+{
+	private final String RAID_COMPLETE_MESSAGE = "Congratulations - your raid is complete!";
+	private final int TOTAL_POINTS = 0, PERSONAL_POINTS = 1, TEXT_CHILD = 4;
+
+	@Inject
+	ChatMessageManager chatMessageManager;
+	@Inject
+	Client client;
+	@Inject
+	RaidsConfig config;
+
+	@Override
+	public void configure(Binder binder)
+	{
+	}
+
+	@Provides
+	RaidsConfig provideConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(RaidsConfig.class);
+	}
+
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (config.pointsMessage())
+		{
+			cacheColors();
+		}
+	}
+
+	@Subscribe
+	public void onChatMessage(ChatMessage event)
+	{
+		if (!config.enabled())
+		{
+			return;
+		}
+
+		if (config.pointsMessage() && event.getType() == ChatMessageType.CLANCHAT_INFO
+				&& event.getMessage().startsWith(RAID_COMPLETE_MESSAGE))
+		{
+			Widget raidsWidget = client.getWidget(WidgetInfo.RAIDS_POINTS_INFOBOX).getChild(TEXT_CHILD);
+			String[] raidPoints = raidsWidget.getText().split("<br>");
+			int totalPoints = Integer.parseInt(raidPoints[TOTAL_POINTS]),
+				personalPoints = Integer.parseInt(raidPoints[PERSONAL_POINTS]);
```

Comment:
dont use variable declaration and assignment like this, just split it to two separate declarations.